debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -311,6 +314,10 @@ private Instant normalize(Timestamp timestamp) {
         return instant;
     }
 
+    public Instant normalizeTimestamp(Timestamp timestamp) {
```

Comment:
Would it make sense to make the existing method public instead?

Reply:
I've made it `protected` as this level of visibility is enough for the method to be called within the same package.